tools/functional-tester/etcd-tester/tester.go

```diff
@@ -41,84 +41,87 @@ func (tt *tester) runLoop() {
 		tt.status.setRound(round)
 		tt.status.setCase(-1) // -1 so that logPrefix doesn't print out 'case'
 		roundTotalCounter.Inc()
+		pfx0 := tt.logPrefix()
 
 		var failed bool
 		for j, f := range tt.failures {
 			caseTotalCounter.WithLabelValues(f.Desc()).Inc()
 			tt.status.setCase(j)
+			pfx1 := tt.logPrefix()
```

Comment:
what the difference between pfx1 and fpx0? i feel they are bad names. we need to be more descriptive.